discovery/github-iterator.go

```diff
@@ -92,16 +99,20 @@ func newGithubClient(token string, timeout time.Duration) *github.Client {
 func (p *GHOrgReposIter) Next(
 	ctx context.Context,
 ) (*github.Repository, time.Duration, error) {
-	if len(p.repos) == 0 {
-		retry, err := p.requestRepos(ctx)
-		if err != nil && len(p.repos) == 0 {
-			return nil, retry, err
+	for {
+		if len(p.repos) == 0 {
+			retry, err := p.requestRepos(ctx)
+			if err != nil && len(p.repos) == 0 {
+				return nil, retry, err
+			}
 		}
-	}
 
-	var next *github.Repository
-	next, p.repos = p.repos[0], p.repos[1:]
-	return next, 0, nil
+		var next *github.Repository
+		next, p.repos = p.repos[0], p.repos[1:]
+		if _, isExcluded := p.excludedRepos[next.GetName()]; !isExcluded {
```

Comment:
not important thing, but usually is idiomatic call the value `isExcluded` just `ok`, sorry for the nitpicky comments

Reply:
Pushed a fix :-p

Don't worry, I do exactly the same :-p